src/busctl/busctl.c

```diff
@@ -1329,13 +1331,20 @@ static int verb_monitor(int argc, char **argv, void *userdata) {
 }
 
 static int verb_capture(int argc, char **argv, void *userdata) {
+        const char info[] =
+                "systemd " STRINGIFY(PROJECT_VERSION) " (" GIT_VERSION ")";
+        _cleanup_free_ char *os = NULL;
+        struct utsname u;
         int r;
 
         if (isatty(fileno(stdout)) > 0)
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Refusing to write message data to console, please redirect output to a file.");
 
-        bus_pcap_header(arg_snaplen, stdout);
+        assert_se(uname(&u) >= 0);
+        asprintf(&os, "%s %s", u.sysname, u.release);
```

Comment:
We prefer to parse os-release - there's a parse_os_release helper, you can call it for `ID` and `VERSION_ID` (second one could be missing). Also there's no formatting, so you can simply `strjoina(id, " ", version_id)` and save the asprintf.

Reply:
ok, this was what wireshark was doing